azure/azure-how-to/instances/create-a-golden-image-pipeline.rst

```diff
@@ -0,0 +1,274 @@
+Create an Ubuntu golden image pipeline
+======================================
+
+This guide will provide instructions for using GitHub Actions to create a pipeline for building Ubuntu "golden" images with the Azure Image Builder (AIB). This guide focuses on automating the use of Azure Image Builder. If you want more information on how to use Azure Image Builder see the :doc:`Create a Pro golden image <create-pro-fips-golden-image>` guide which explains the individual steps.
+
+What you'll need
+----------------
+
+- A Microsoft Azure account
+- `Azure Command-Line Interface`_
+- A GitHub account
+- (Optional) Golang for local debugging/testing
+
+Set up your Azure credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To authenticate with the Azure Go SDK you are going to use an Azure service principal with a client secret.
+For help creating an Azure service principal see `Create an Azure service principal`_. Make sure to assign this service principal the owner role if you don't want to manually create and setup the managed identity needed by Azure Image Builder.
+
+You will need the following to authenticate with Azure in later steps:
+
+- Your Azure subscription ID
+- Your Azure tenant ID
+- Your service principal client ID
+- Your service principal client secret
+
+Setup your GitHub repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To make use of GitHub Actions you will need to create a new repository. After creating the repository you'll need the URL of the form ``git@github.com:<username>/<repository-name>.git`` for SSH based authentication or ``https://github.com/<username>/<repo-name>.git`` for HTTPS authentication.
+
+Clone the sample repository `azure-image-builder-pipeline-demo`_ and set the remote URL to your newly created repository.
+
+.. code::
+
+   git clone https://github.com/canonical/azure-image-builder-pipeline-demo.git
+   git remote set-url origin <your Git repository URL>
+   git push -u origin main
+
+.. note::
```

Comment:
Let's move this note to a separate 'Local setup' section at the end of this (GitHub) section.